### ISSUE TITLE
Experimental index

### ISSUE THREAD

**Issue by liyuzhan** [zeonchen/SFMGTL#2]
Why is the index running according to the code you uploaded poor? Is it because some of the parameters are set incorrectly?

**Comment by zeonchen** [comment]
Hi, can you provide more info related to your setting?

**Comment by liyuzhan** [comment]
Just your GitHub code and run instructions, run pickup and dropoff, take the average. But the results are far from yours. Such as 
python main.py --scity 'NY' --tcity 'DC' --datatype 'pickup' --batch_size 64 --seed 12 --train_days 1 --hidden_dim 32
python main.py --scity 'NY' --tcity 'DC' --datatype 'dropoff' --batch_size 64 --seed 12 --train_days 1 --hidden_dim 32

**Comment by zeonchen** [comment]
For 1-day training, please reduce the training epoch to 10 or 15 epochs due to overfitting. For 3-day and 7-day training, the current setting is okay.

**Comment by liyuzhan** [comment]
Are there any other parameters you need to set in your code? 1 day, 3 day, and 7 day results are not very good

**Comment by zeonchen** [comment]
Please post your settings and results with screenshots for more investigation.

**Comment by liyuzhan** [comment]
The following are the relevant parameters and code examples for reproducing the bicycle demand with NY as the source city and DC as the target city. Here, I changed the training from 50 to 15. The indicators of RMSE and R^2 are significantly improved, but the corresponding MAE indicators become worse.
[reproduction.docx](https://github.com/user-attachments/files/16940739/reproduction.docx)

**Comment by zeonchen** [comment]
I just re-run the code with your settings and find the pickup and dropoff metrics as follows:
![62d1c5fc1670f9a03273c0f838ee86e](https://github.com/user-attachments/assets/a3777280-0979-4cdf-884f-b070d6393efe)

![bf16696664fda77131015a3210e4f78](https://github.com/user-attachments/assets/c5fc2878-6d67-4315-9fae-568af70333fe)

Both results are better than yours. I believe the discrepency of the running environment and hardware lead to the differences and the current codes are not that robust. We will modify the codes later to make it more stable. Thanks for your feedback.
